gradio/exceptions.py

```diff
@@ -57,7 +58,7 @@ class GradioVersionIncompatibleError(Exception):
 
 
 @document(documentation_group="modals")
-class Error(Exception):
+class Error(AppError):
```

Comment:
Nit but I think AppError should subclass `Exception` instead of `ValueError` ?

Reply:
I think if you're trying to catch AppErrors in your client code, you would catch AppError explicitly as opposed to catching ValueErrors generally but sound good it's a minor point!